scripts/snakeeyes.coffee

```diff
@@ -0,0 +1,5 @@
+# snakeeyes is back ... and this time it's personal!
+
+module.exports = (robot) ->
+  robot.hear /snakeeyes/i, (msg) ->
```

Comment:
Not sure if you'd want this, but you could use `/snake ?eyes/i` to have Brehm reply to both "snakeeyes" and "snake eyes".